.nvmrc

```diff
@@ -0,0 +1 @@
+4.3
```

Comment:
Should this be 6.10?

Reply:
I'll defer to @bilts, I copied this out of his sled code.